src/SensioLabs/Melody/Tests/IntegrationTest.php

```diff
@@ -48,6 +48,13 @@ public function testRunWithCache()
         $this->assertSame('Hello world', $output);
     }
 
+    public function testRunWithConstraints()
+    {
+        $this->melodyRun('hello-world-with-constraints.php');
+        $output = $this->melodyRun('hello-world-with-constraints.php');
```

Comment:
Suppose it was for gache testing, it's similar to: https://github.com/sensiolabs/melody/blob/master/src/SensioLabs/Melody/Tests/IntegrationTest.php#L44